docs/CHANGELOG.asciidoc

```diff
@@ -37,6 +37,7 @@
 * Reduce the peak memory used by boosted tree training and fix an overcounting bug
 estimating maximum memory usage. (See {ml-pull}781[#781].)
 * Stratified fractional cross validation for regression. (See {ml-pull}784[#784].)
+* Added `geo_point` supported output for `lat_long` function records. (See {ml-pull}809[#809].)
```

Comment:
Because of the title of elastic/elasticsearch#47050, as things stand this release note will appear in addition to "Add new geo_results.(actual|typical) fields for lat_lon results".  As a user I would wonder whether these two things were the same or different enhancements, so it's probably best to have just one of them.

Since this changelog is more flexible and can reference two PRs, it's probably best to mark elastic/elasticsearch#47050 as a non-issue so that it doesn't get auto-generated into the release notes, but then mention it in this one too, using `(See {ml-pull}809[#809], {pull}47050[#47050].)`.